packages/toolkit/src/utils/appValidation.ts

```diff
@@ -30,7 +34,21 @@ function findFunctionSignatureCollisions(abi1: AbiItem[], abi2: AbiItem[]) {
   return collisions
 }
 
-export function checkSignatureCollisionsWithProxy(abi: AbiItem[]) {
+function checkAssignedGlobalVariables(sourceCode: string) {
+  const assignments = parseGlobalVariableAssignments(sourceCode)
+  const assignments2 = parseConstructorAssignments(sourceCode)
```

Comment:
Since the constructor of an app is never called, we could perhaps simply check if the code contains the constructor and output a warning instead of verifying its assignments. 
See https://hack.aragon.org/docs/aragonos-building#constructor-and-initialization for more details about app initialization.

Reply:
Makes sense. Fixed!